data/json/speech.json

```diff
@@ -2186,7 +2192,26 @@
   {
     "type": "speech",
     "speaker": "foodperson_mask",
-    "sound": "\"FoodPlace: It's the Calories.\"",
+    "sound": "\"At FoodPlace your excellent value brings us nourishment and delight.\"",
+    "volume": 15
+  },
+  {
+    "type": "speech",
+    "speaker": "foodperson_mask",
+    "sound": "\"At FoodPlace our food is not the problem!\"",
```

Comment:
This might be a bit too suspicious, other slogans are weirdly bland and unspecific, this one kind of suggest that the food could be unhealthy.

Reply:
This Calories one isn't part of my PR - it was there before.

Want to remove it? I'm fine with that. Or change it to something like "FoodPlace: It's the place with the food."

Reply:
I changed "At FoodPlace our food is not the problem!" to something similar, but less pejorative.

Leaving this suggestion open in case you want to modify "It's the Calories" - I think that one is from Erk, but not positive.

Reply:
> This Calories one isn't part of my PR - it was there before


I think there's a mix up of comments here. I meant that this `At FoodPlace your excellent value brings us nourishment and delight.\"` seem to imply that Foodplace eats people.

Reply:
ok great - already fixed that